parsers/jscript.c

```diff
@@ -126,7 +126,8 @@ enum eKeywordId {
 	KEYWORD_sap,
 	KEYWORD_return,
 	KEYWORD_class,
-	KEYWORD_extends
+	KEYWORD_extends,
+	KEYWORD_static
```

Comment:
Let's put  a comma (`,`) for the last element:
```
KEYWORD_static,
```

C language allows this.  When someone wants to add more elements in the future, the last `comma` will make the changes one line smaller. Smaller diff makes previewers happy:-)